packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -192,6 +196,65 @@ describeNoCompat("Summaries", (getTestObjectProvider) => {
 		assert(ackNackResult.success, "summary op should be acked");
 	});
 
+	it("sample use for customized containerRuntimeWithBlob", async () => {
+		const blobContents = "sample payload";
+
+		// sample API to load a containerRuntime which has extra blob in the summary
+		async function containerRuntimeWithBlob(params: IContainerRuntimeParams) {
+			// Blob path in container snapshots
+			const path = ["loop", "guestComponents"];
+
+			const extraBlob = await loadRuntimeBlob(params, path);
```

Comment:
Why is this step needed during container runtime creation? extraBlob is not even used.